File: Swift-C++-IR-Comparison/test/controlFlow/Swift/ForInLoopDict.swift
@main
struct MyMain {

    static func main() {
        var a = [1: 2, 2: 3, 3: 4, 4: 5, 5: 6]
        var sum = 0
        for (b, c) in a {
            sum = sum + b + c
        }
    }
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-0/ForInLoopDict.cpp

 #include <iostream>
 #include <map>
 
int main() 
{
    std::map<int, int> a = { {1,2}, {2,3}, {3,4}, {4,5}, {5,6} };
    int sum = 0;
 
    for (const auto& b : a ) 
    {
            sum += (b.first + b.second);
    }
}